### PR TITLE
Fixing more URL and Code instructions.

### PR DESCRIPTION
**Pull request by HideLord**
Fixing more URL and Code instructions. Something I noticed is that code instructions are incredibly repetitive (at least 7-8 'add two numbers' and just as much Fibonacci, factorial and sort instructions). They are also very simplistic.

### CODE REVIEW

File: alpaca_data_cleaned.json [hunk 1]
```diff
@@ -240265,9 +240265,9 @@
         "output": "GDP, inflation, trade, employment, fiscal policy, taxation, budget deficit, investments, growth, recession."
     },
     {
-        "instruction": "Given a website URL, generate four questions about the website.",
+        "instruction": "Generate 4 questions for a beginner software position interview.",
         "input": "www.example.com",
```
Comment (gururise) [inline alpaca_data_cleaned.json:240269]: I think this input should be blank.

File: alpaca_data_cleaned.json [hunk 2]
```diff
@@ -242565,9 +242565,9 @@
         "output": "The two datasets have similarities in that both contain the number 3. They differ in that the first dataset includes numbers 9, 8, 11, and 0, while the second dataset includes numbers 10, 2, 12, and 5."
     },
     {
-        "instruction": "Turn the following string into a valid URI:",
+        "instruction": "URI vs URL.",
         "input": "https:\\\\www.example.com",
```
Comment (gururise) [inline alpaca_data_cleaned.json:242569]: Another input that should be empty.

**Review by gururise — CHANGES_REQUESTED** [review]
found two small issues.

File: alpaca_data_cleaned.json (hunk 2 above)
Comment (HideLord) [reply alpaca_data_cleaned.json:242569]: Fixed

File: alpaca_data_cleaned.json (hunk 1 above)
Comment (HideLord) [reply alpaca_data_cleaned.json:240269]: Fixed

**Review by gururise — APPROVED** [review]
Thanks!

**Pull request merged**